packages/react-components/priority-overflow/src/useOverflowContainer.ts

```diff
@@ -11,10 +11,10 @@ import type {
   OverflowDividerEntry,
   OverflowManager,
   ObserveOptions,
-} from '@fluentui/priority-overflow';
+} from './';
```

Comment:
we should not move react hooks to this package - it is intended for vanilla js code only